server/utils/validations/exMoneyValidation.js

```diff
@@ -0,0 +1,19 @@
+const yup = require('yup');
+
+const schema = yup.object().shape({
+  amount: yup
+    .number()
+    .required('Please Enter your amount you want to exchange')
+    .positive(),
+  from: yup
+    .string()
+    .required('Please Enter currency you want to exchange from'),
+  to: yup.string().required('Please Enter currency you want to exchange to it'),
+});
+
+const exchangeMoneyValidation = (transactions) =>
+  schema.isValid({
+    transactions,
+  });
```

Comment:
change transaction word to be `newConversion` here , please focus in everything while coding
```suggestion
const exchangeMoneyValidation = (newConversion) =>
  schema.isValid({
    newConversion,
  });
```